app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    # @user = User.find(params[:id])
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable, It's okay for the actions to be empty for now. 😉